cmd/argocd/commands/app.go

```diff
@@ -404,9 +403,13 @@ func setAppOptions(flags *pflag.FlagSet, app *argoappv1.Application, appOpts *ap
 		case "revision":
 			app.Spec.Source.TargetRevision = appOpts.revision
 		case "values":
-			setHelmOpt(&app.Spec.Source, appOpts.valuesFiles, nil)
+			setHelmOpt(&app.Spec.Source, helmOpts{valueFiles: appOpts.valuesFiles})
 		case "release-name":
-			setHelmOpt(&app.Spec.Source, nil, &appOpts.releaseName)
```

Comment:
not sure why this had a pointer to a string rather than just a string - thoughts?

Reply:
I suspect it was just in case. Nil should be no different from setting an empty string